app/src/main/java/com/sunasterisk/movieintroduction/data/source/MovieRepository.kt

```diff
@@ -0,0 +1,17 @@
+package com.sunasterisk.movieintroduction.data.source
+
+import com.sunasterisk.movieintroduction.data.model.ResponseGenre
+import com.sunasterisk.movieintroduction.data.source.remote.HomeData
+
+class MovieRepository private constructor(private var remoteGenre: MovieDataSource.Remote ): MovieDataSource.Remote{
```

Comment:
Chú ý class header 
https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting
```suggestion
class MovieRepository private constructor(
    private val remoteGenre: MovieDataSource.Remote 
): MovieDataSource.Remote{
```